client/service/dashboard_client.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=MHP DAShboard client
+
+[Install]
+WantedBy=default.target
+
+[Service]
+Environment=NODE_PORT=3000
+Restart=on-failure
+# Note for this to work we need to run the below command from the DAShboard repository client directory, the Working directory is set by install.sh script
+# Ensure serve is downloaded: `yarn global add serve`
```

Comment:
This is important, as `serve` is not installed by default. Worth moving to the readme.